front_end/webserver.py

```diff
@@ -1106,9 +1106,11 @@ def post(self, course, assignment, exercise):
                                     tests_dict = []
                                     for i in range(len(tests)):
                                         tests_dict.append({**tests[i], **exercise_details["tests"][i]})
```

Comment:
Please modify this code, too, to be consistent with dictionary syntax we are using elsewhere.